resources/views/pages/billing/index.blade.php

```diff
@@ -452,6 +452,13 @@
             </div>
          </div>
       </div>
+      <div class="col-12 col-md-12 col-xl-6">
+         <div class="card">
+             <div style="display: {{ $svgDisplay }};">
+                 {!! $svg !!}
+            </div>
```

Comment:
I'd recommend hiding the col container. If the SVG is not displayed, this leaves the card container showing a border (as a line since there's no content) and the col container would probably also take up space and push future possible blocks to the side.

![image](https://github.com/SonarSoftwareInc/customer_portal/assets/123477228/a8c5671e-9ee4-46c9-87c2-e98eab94044f)